Lib/test/test_re.py

```diff
@@ -2190,6 +2190,17 @@ class ImplementationTest(unittest.TestCase):
     Test implementation details of the re module.
     """
 
+    @cpython_only
+    def test_immutable(self):
```

Comment:
Please add a comment to explain the purpose of the test, something like:

"bpo-43908: check that re types are immutable"